src/main/scala/akka/contrib/stream/pattern/reconnect/ReconnectingStreamHttp.scala

```diff
@@ -33,7 +31,7 @@ object ReconnectingStreamHttp {
       interval: FiniteDuration,
       retriesRemaining: Long,
       settings: Option[ClientConnectionSettings],
-      onConnection: (ConnectionStatus => Unit)) extends ReconnectionData[HttpConnected] {
+      onConnection: (HttpConnected => Unit)) extends ReconnectionData[HttpConnected] {
```

Comment:
This is exactly what we have to move away from here. The instance may not be a success.